libraries/chain/steem_evaluator.cpp

```diff
@@ -95,16 +95,10 @@ void witness_update_evaluator::do_apply( const witness_update_operation& o )
       wlog( "Wrong fee symbol in block ${b}", ("b", _db.head_block_num()+1) );
    }
 
-   #pragma message( "TODO: This needs to be part of HF 20 and moved to validate if not triggered in previous blocks" )
-   if( _db.is_producing() )
+   // May be we can move this check to - validate.hpp
+   if( _db.is_producing() && _db.has_hardfork( STEEM_MAX_BLOCK_SIZE_HF20 ))
```

Comment:
What you're saying here is when Steem has hardfork 2097152 then enforce the rule. We want to enforce this on HF 20.